CI failed on master

https://app.wercker.com/#buildstep/57274f6d5b2d079422057c9e

```
  1) StatsSubscription #update records current stats to statsLogs:

      AssertionError:   # test/lib/subscriptions/subscription/stats-test.js:37
  
  assert.deepEqual(sub.statsLogs[item.id], { timestamps: [_.now()], stats: _.mapValues(_.keyBy(item.stats, "key"), stat => [stat.value]) })
                   |   |        ||    |    |             || |              | |         | |     |    |                                      
                   |   |        ||    |    |             || |              | |         | |     |    [Object{key:"point",value:17371,label:"Point",icon:"plus",unit:"pt",link:"http://syosetu.com/usernovelmanage/top/ncode/54104/"},Object{key:"ratePoint",order:2,value:833,label:"Rate Point",icon:"star",unit:"pt"},Object{key:"bookmarkCount",order:3,value:90,label:"Bookmarks",icon:"bookmark"}]
                   |   |        ||    |    |             || |              | |         | |     Object{id:"n26ab",title:"Test novel 26",url:"http://ncode.syosetu.com/n26ab/",type:"novel",body:"Test novel description 26\nHello, world!",authorName:"Author26",authorUrl:"http://mypage.syosetu.com/2617845296922624/",createdAt:1467826711436,updatedAt:1467826712349,links:[#Object#,#Object#],stats:[#Object#,#Object#,#Object#]}
                   |   |        ||    |    |             || |              | |         | Object{point:Object{key:"point",value:17371,label:"Point",icon:"plus",unit:"pt",link:"http://syosetu.com/usernovelmanage/top/ncode/54104/"},ratePoint:Object{key:"ratePoint",order:2,value:833,label:"Rate Point",icon:"star",unit:"pt"},bookmarkCount:Object{key:"bookmarkCount",order:3,value:90,label:"Bookmarks",icon:"bookmark"}}
                   |   |        ||    |    |             || |              | |         #function#                                          
                   |   |        ||    |    |             || |              | Object{point:[17371],ratePoint:[833],bookmarkCount:[90]}      
                   |   |        ||    |    |             || 1462461670269  #function#                                                      
                   |   |        ||    |    |             |#function#                                                                       
                   |   |        ||    |    |             [1462461670269]                                                                   
                   |   |        ||    |    Object{timestamps:[1462461670269],stats:Object{point:#Array#,ratePoint:#Array#,bookmarkCount:#Array#}}
                   |   |        ||    "n26ab"                                                                                              
                   |   |        |Object{id:"n26ab",title:"Test novel 26",url:"http://ncode.syosetu.com/n26ab/",type:"novel",body:"Test novel description 26\nHello, world!",authorName:"Author26",authorUrl:"http://mypage.syosetu.com/2617845296922624/",createdAt:1467826711436,updatedAt:1467826712349,links:[#Object#,#Object#],stats:[#Object#,#Object#,#Object#]}
                   |   |        Object{timestamps:[1462212664469,1462282095275,1462360634327,1462461670269],stats:Object{point:#Array#,ratePoint:#Array#,bookmarkCount:#Array#}}
                   |   Object{"n26ab":Object{timestamps:#Array#,stats:#Object#},"n27ab":Object{timestamps:#Array#,stats:#Object#},"n28ab":Object{timestamps:#Array#,stats:#Object#},"n29ab":Object{timestamps:#Array#,stats:#Object#},"n30ab":Object{timestamps:#Array#,stats:#Object#},"n31ab":Object{timestamps:#Array#,stats:#Object#}}
                   StatsSubscription{data:Object{type:"stats",feedUrl:"novelous-feed://narou/novels",feedData:#Object#,enabled:true,lastUpdatedAt:1462461670269,statsLogs:#Object#},_feed:Feed{data:#Object#},_feedFetcher:FetcherNarouNovels{}}
  
      + expected - actual

       {
         "stats": {
           "bookmarkCount": [
      -      2600934332956672
      -      5107107221209088
      -      1023656742354944
             90
           ]
           "point": [
      -      8329524546633728
      -      4758183547502592
      -      6928243177291776
             17371
           ]
           "ratePoint": [
      -      8990025276456960
      -      7738900143407104
      -      2065640630255616
             833
           ]
         }
         "timestamps": [
      -    1462212664469
      -    1462282095275
      -    1462360634327
           1462461670269
         ]
       }
```

--- a/test/lib/subscriptions/subscription/stats-test.js
+++ b/test/lib/subscriptions/subscription/stats-test.js
@@ -27,16 +27,20 @@ describe("StatsSubscription", () => {
   describe("#update", () => {
     it("records current stats to statsLogs", async () => {
       const item = sub.feed.items[0];
+      const originalLog = _.cloneDeep(sub.statsLogs[item.id]);
+
       const [newFeed, newFeedItem] = helpers.getFeedWithNewItem(sub.feed, "novelFeedItem");
       helpers.stubFetchFeed(sub, newFeed);
-      sinonsb.stub(_, "now").returns(newFeedItem.updatedAt);
+      sinonsb.stub(_, "now").returns(Math.max(item.updatedAt, newFeedItem.updatedAt));
 
       await sub.update();
 
       assert(sub.feed === newFeed);
       assert.deepEqual(sub.statsLogs[item.id], {
-        timestamps: [_.now()],
-        stats: _.mapValues(_.keyBy(item.stats, "key"), stat => [stat.value]),
+        timestamps: originalLog.timestamps.concat([_.now()]),
+        stats: _.mapValues(_.keyBy(item.stats, "key"), (stat, key) => {
+          return originalLog.stats[key].concat([stat.value]);
+        }),
       });
       assert.deepEqual(sub.statsLogs[newFeedItem.id], {
         timestamps: [_.now()],

--- a/test/test-utils/factory.js
+++ b/test/test-utils/factory.js
@@ -23,13 +23,13 @@ factory.builder = (n, name) => {
 };
 
 factory.timestamp = (base = TIMESTAMP_BASE) => {
-  return () => base + _.random(0, 365 * 24 * 60 * 60 * 1000);
+  return () => base + _.random(0, 24 * 60 * 60 * 1000);
 };
 
 factory.timestamps = (n, base = TIMESTAMP_BASE) => {
   return () => {
     let time = base;
-    return _.times(n, () => (time += _.random(1, 24 * 60 * 60 * 1000)));
+    return _.times(n, () => (time += _.random(1, 60 * 60 * 1000)));
   };
 };
 

